wsmaster/che-core-api-devfile/pom.xml

```diff
@@ -1,184 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<!--
-
-    Copyright (c) 2012-2018 Red Hat, Inc.
-    This program and the accompanying materials are made
-    available under the terms of the Eclipse Public License 2.0
-    which is available at https://www.eclipse.org/legal/epl-2.0/
-
-    SPDX-License-Identifier: EPL-2.0
-
-    Contributors:
-      Red Hat, Inc. - initial API and implementation
-
--->
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-    <modelVersion>4.0.0</modelVersion>
-    <parent>
-        <artifactId>che-master-parent</artifactId>
-        <groupId>org.eclipse.che.core</groupId>
-        <version>7.0.0-beta-6.0-SNAPSHOT</version>
-    </parent>
-    <artifactId>che-core-api-devfile</artifactId>
```

Comment:
Looks like they're still 3 places where this artifact is used. Build works locally only because of existence outdated artifact in local maven repo. Please make sure that it is not used anymore otherwise build after next release will fail
![Screenshot_20190528_144548](https://user-images.githubusercontent.com/5887312/58475671-4c139380-8157-11e9-93c3-3c08c3dbf29a.png)

Reply:
Also, please update CODEOWNERS

https://github.com/eclipse/che/blob/3d3f4a463cc02ee6af5cacda93df6cf48e3d0413/.github/CODEOWNERS#L51